src/generated/api/src/model.rs

```diff
@@ -7157,7 +7157,7 @@ impl wkt::message::Message for SystemParameter {
 #[non_exhaustive]
 pub struct Usage {
     /// Requirements that must be satisfied before a consumer project can use the
-    /// service. Each requirement is of the form <service.name>/<requirement-id>;
+    /// service. Each requirement is of the form <service.name>/&lt;requirement-id&gt;;
```

Comment:
Hm interestingly, even though <service.name> isn't escaped, it still renders as:

<img width="1017" alt="Screenshot 2025-01-22 at 10 52 36 PM" src="https://github.com/user-attachments/assets/ecbc9470-711b-4584-b563-59c8997b32b7" />